.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,7 +3,6 @@
 
 ## Checklist ##
 ### Essentials ###
-- [ ] Passed code style checking (`make lint`)
```

Comment:
Why is this removed?

Reply:
I've talked to a couple of people and we are checking this in ci, what's the point of running it locally?

Reply:
Since there was a bottleneck for scheduling and running tasks on old CI, each failed attempt even for sanity test meant wasted time, which was the motivation to add it here. You can propose a change to this with the updated status of the CI and your motivation.

May I request that you put this proposal either in an email to dev-list, or in a separate PR, so that only changes that are relevant in the notes and PR title are included?

Reply:
@szha ok